packages/anoncreds/src/AnonCredsModule.ts

```diff
@@ -38,6 +38,20 @@ export class AnonCredsModule implements Module {
     dependencyManager.registerSingleton(AnonCredsLinkSecretRepository)
   }
 
+  public async initalize(agentContext: AgentContext) {
+    if (this.config.autoCreateLinkSecret === false) {
+      return
+    }
+
+    const api = agentContext.dependencyManager.resolve(AnonCredsApi)
```

Comment:
I'd call this variable `anoncredsApi` just to be consistent with other parts of the framework where apis are resolved.

Reply:
Fixed